MobileCenterCrashes/MobileCenterCrashes/Internals/Util/MSCrashesUtil.m

```diff
@@ -90,24 +93,31 @@ + (NSURL *)logBufferDir {
 
 + (NSURL *)wrapperExceptionsDir {
```

Comment:
If all three functions in this class share the same implementations, would you refactor them to make the code easier to understand?